components/router/route.js

```diff
@@ -33,10 +33,30 @@ class Route {
 class GamefaceRoute extends HTMLElement {
     constructor() {
         super();
-
         this.onClick = this.onClick.bind(this);
+        this.onHistoryChange = this.onHistoryChange.bind(this);
 
-        
+        window.removeEventListener('onHistoryChange', this.onHistoryChange);
+        window.addEventListener('onHistoryChange', this.onHistoryChange);

```

Comment:
Why we remove and add event listener after that?
Are we supposed to add event listener inside `connectedCallback` and remove it inside `disconnectedCallback`?